scripts/logger_pc_app/README.md

```diff
@@ -0,0 +1,123 @@
+# ZephyrLogParser
```

Comment:
Can we please write this as a zephyr extension instead? I don't think we need yet another executable/Python script. CC @mbolivar

Reply:
Discussed offline - rewriting as a Zephyr extension will not be a part of this PR.